packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -285,6 +286,36 @@ export class MetriportMedicalApi {
     return resp.data;
   }
 
+  /**
+   * Returns the URL for a medical record summary, if it exists.
+   *
+   * @param patientId The ID of the patient whose data is to be returned.
+   * @param conversionType Indicate how the medical record should be rendered. Can be "html" or "pdf".
+   * @return The URL for the medical record summary.
+   */
+  async getMedicalRecordSummary(
+    patientId: string,
+    conversionType: "html" | "pdf"
+  ): Promise<{ url: string }> {
```

Comment:
You might be following the pattern of `getDocumentUrl()`, but as a client of the API, it would be easier to just get the URL:

```ts
const url = await getMedicalRecordSummaryURL(...);
```

Note: I'm just referring to the SDK response, I think the API's response payload should be `{ url: string }` 👍